gatsby-node.js

```diff
@@ -34,7 +34,7 @@ exports.createPages = ({ graphql, boundActionCreators }) => {
           path: edge.node.slug, // required
           component: blogPost,
           context: {
-            slug: edge.node.slug,
+            slug: { eq: edge.node.slug },
```

Comment:
this isn't great :/ this is a limitation/constraint Relay has you can't use nested arguments in a query input like: ` markdownRemark(slug: { eq: $slug })`